backend/app/utils.py

```
@@ -14,104 +1,101 @@
             return {
                 "google_id": id_info["sub"],
                 "email": id_info["email"],
                 "email_verified": id_info.get("email_verified", False),
                 "full_name": id_info.get("name"),
             }
-        except ValueError as e:
-            raise HTTPException(
-                status_code=400, detail=f"Token verification failed: {str(e)}"
-            )
+        except ValueError:
+            raise HTTPException(status.HTTP_400_BAD_REQUEST, detail=INVALID_TOKEN)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)